recipes/prefect-ray/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "prefect-ray" %}
+{% set version = "0.2.0.post2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1e23a654fc599a41f8c6bbb8995a5adc58d9a458038ba9ddb4321fefb5540cd4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  # ray does not have a recipe for Mac
+  skip: true  # [osx,py<37]
```

Comment:
Normally this wouldn't be a problem - a noarch package will only be available for the platforms that its dependencies are available, and just the mac build would fail here (but not on the feedstock). However, it looks like pickle5 is _only_ available for 3.7, so you need the selector, so this does look like it needs to be here (but not for this stated reason).